osquery/tables/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ endif()
 # TODO: These tables should be made Windows-compatible
 if(NOT WINDOWS)
   file(GLOB OSQUERY_POSIX_NETWORKING_TABLES "networking/posix/*.cpp")
-  file(GLOB OSQUERY_CROSS_FORENSIC_TABLES "forensic/*.cpp")
+  file(GLOB OSQUERY_POSIX_FORENSIC_TABLES "forensic/posix/*.cpp")
```

Comment:
This is done for sleuthkit libs, which I didn't think were ready for Windows yet, but that's (i believe) only blocked by us having the libs built and available.